app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt

```diff
@@ -74,9 +74,9 @@ internal fun multiColoredExampleCalendarColoring(
   }
   return CalendarColoring(
     setOf(
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkRed200), redSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkYellow100), yellowSet),
-      ColoredBucket(ContextCompat.getColor(context, R.color.bpkGreen100), greenSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkNara), redSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkBagan), yellowSet),
+      ColoredBucket(ContextCompat.getColor(context, R.color.bpkSegano), greenSet),
       ColoredBucket(ContextCompat.getColor(context, R.color.bpkGray50), greySet),
       ColoredBucket(null, emptySet, ContextCompat.getColor(context, R.color.bpkPink500))
```

Comment:
I think we need to use some of the new pinks here.

Reply:
Updated according to this https://app.zeplin.io/project/5d6fa72570db2f18bdf8e209/screen/5d6fb58fc1b83b981a512b5e